serverlessworkflow/modules/ROOT/pages/release_notes.adoc

```diff
@@ -1,21 +1,14 @@
 = New features on {page-component-display-version}
 :compat-mode!:
 
-== Notable changes
+* New default diagram canvas (currently only available in VSCode extension)
```

Comment:
@danielzhe I think this is not true. The new stunner editor is presented in all channels (VSCode, sandbox, online, chome, etc). The only thing is is only VSCode specific is the setting to swith back to Mermaid editor.
Can you please @ederign @paulovmr confirm this?
Thx in advance!

Reply:
Please, keep the release notes unchanged here. I'll open a new PR to update the release notes targeting 1.28 of Kogito, and we can also add a new sub-title for the editors. In that case, please keep the format the same as we have here with a linked KOGITO Jira targeting the Editor's version compatible with Kogito 1.28.

Reply:
@romartin correct. Except for Chrome Extension (@paulovmr will send a PR for it), all channels are default. Both on VS Code and Web Tools (kubesmarts) there is a way to switch back to mermaid.

Reply:
@danielzhe could you please fix this ?

Reply:
I reverted this change as requested by @ricardozanini in the comment bellow.